cirrus/server/cirrus/feature_manifest.py

```diff
@@ -1,13 +1,42 @@
+import json
+import logging
 from typing import Any, Dict, List
 
+from fml_sdk import FmlClient, FmlError  # type: ignore
+
+from .settings import channel, fml_path
+
+logger = logging.getLogger(__name__)
+
 
 class FeatureManifestLanguage:
+    def __init__(self):
+        self.fml_client = FmlClient(fml_path, channel)
+        self.merge_errors: List[FmlError] = []
+
     def compute_feature_configurations(
         self,
         enrolled_partial_configuration: Dict[str, Any],
-        feature_configurations: List[Dict[str, Any]],
-    ) -> Dict[str, str]:
-        return {"feature": "test"}
+    ) -> Dict[str, Any]:
+        feature_configs = {
+            key: value["feature"]["value"]
+            for key, value in enrolled_partial_configuration[
+                "enrolledFeatureConfigMap"  # slug, featureid, value,
+            ].items()
+        }
+        merged_res: Dict[str, Any] = self.fml_client.merge(  # type: ignore
+            feature_configs
+        )
```

Comment:
`merged_res` is not a Dict according to the [UDL](https://github.com/mozilla/application-services/blob/b8d3b225ee6e7069ff4e2c14e8099d9514c9f034/components/support/nimbus-fml/src/fml.udl#L11)

Reply:
Yep, you should be able to import a `MergedJsonWithErrors` classref from `fml_sdk`

Reply:
You should be able to leave it without a type annotation and it should infer the type from the return of self.fml_client.merge

Reply:
it doesn't infer that, `strict` mode needs to be defined with the type

Reply:
good call @jeddai and @brennie

Reply:
changed with the correct type